inc/azure_c_util/thandle.h

```diff
@@ -12,7 +12,10 @@
 #include "refcount_os.h"
 
 /*the incomplete unassignable type*/
-#define THANDLE(T) const T* const
+#define THANDLE(T) MU_C2(CONST_P2_CONST_,T)
```

Comment:
>T) MU_C2(CONST_P2_CONST_,T) [](start = 16, length = 27)

Ah, this should fix the issue I mentioned with passing the array of handles as out arg.

Reply:
Nice.

---
In reply to: [392014336](https://github.com/Azure/azure-c-util/pull/20#discussion_r392014336) [](ancestors = 392014336)